indy_common/types.py

```diff
@@ -368,6 +368,19 @@ class ClientAuthRuleOperation(MessageValidator):
     )
 
 
+class ClientGetAuthRuleOperation(MessageValidator):
+    schema = (
+        (TXN_TYPE, ConstantField(GET_AUTH_RULE)),
+        (AUTH_ACTION, ChooseField(values=(ADD_PREFIX, EDIT_PREFIX), optional=True)),
+        (AUTH_TYPE, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT, optional=True)),
+        (FIELD, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT, optional=True)),
+        (OLD_VALUE, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT,
```

Comment:
OLD_VALUE can be empty now (see https://github.com/hyperledger/indy-node/pull/1202)